### ISSUE TITLE
 Error in loading item ytsearch:(non-specific track)

### ISSUE THREAD

**Issue by XenoMax66** [sedmelluq/lavaplayer#509]
Searching for most non-specific(?) songs now throws `Error in loading item`.  The track I was searching for was "Relaxing Music".
This is something that's only started happening since the past month or two.

This goes for most non-specific track/music/song searching, such as: "driving music", "Study Music", "space music", "ambient music", etc. 

Searching for specific song names such as: "Astral Hammer"/"fluffing a duck"/"Flight Hymn" etc,  still works. 

I am using the latest version of lavaplayer `<version>1.3.50</version>` and lavaplayer extras `<version>1.3.13</version>` and using the `bintray-sedmelluq` repository.

```[14:41:45] [INFO] [JDA]: Login Successful!
[14:41:45] [INFO] [WebSocketClient]: Connected to WebSocket
[14:41:46] [INFO] [JDA]: Finished Loading!
[14:42:10] [ERROR] [DefaultAudioPlayerManager]: Error in loading item ytsearch:relaxing music
com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Something went wrong when looking up the track
	at com.sedmelluq.discord.lavaplayer.tools.ExceptionTools.wrapUnfriendlyExceptions(ExceptionTools.java:43)
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.dispatchItemLoadFailure(DefaultAudioPlayerManager.java:237)
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$createItemLoader$0(DefaultAudioPlayerManager.java:226)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at com.sedmelluq.discord.lavaplayer.tools.OrderedExecutor$ChannelRunnable.executeQueue(OrderedExecutor.java:98)
	at com.sedmelluq.discord.lavaplayer.tools.OrderedExecutor$ChannelRunnable.run(OrderedExecutor.java:87)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.RuntimeException: java.lang.NullPointerException
	at com.sedmelluq.discord.lavaplayer.tools.ExceptionTools.wrapUnfriendlyExceptions(ExceptionTools.java:57)
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeSearchProvider.loadSearchResult(YoutubeSearchProvider.java:74)
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager$LoadingRoutes.search(YoutubeAudioSourceManager.java:254)
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager$LoadingRoutes.search(YoutubeAudioSourceManager.java:220)
	at com.sedmelluq.discord.lavaplayer.source.youtube.DefaultYoutubeLinkRouter.route(DefaultYoutubeLinkRouter.java:34)
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager.loadItemOnce(YoutubeAudioSourceManager.java:188)
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager.loadItem(YoutubeAudioSourceManager.java:127)
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.checkSourcesForItemOnce(DefaultAudioPlayerManager.java:437)
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.checkSourcesForItem(DefaultAudioPlayerManager.java:419)
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$createItemLoader$0(DefaultAudioPlayerManager.java:218)
	... 6 common frames omitted
Caused by: java.lang.NullPointerException: null
	at com.sedmelluq.discord.lavaplayer.tools.DataFormatTools.durationTextToMillis(DataFormatTools.java:91)
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeSearchProvider.extractPolymerData(YoutubeSearchProvider.java:161)
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeSearchProvider.lambda$polymerExtractTracks$0(YoutubeSearchProvider.java:149)
	at java.util.ArrayList.forEach(Unknown Source)
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeSearchProvider.polymerExtractTracks(YoutubeSearchProvider.java:148)
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeSearchProvider.extractSearchResults(YoutubeSearchProvider.java:94)
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeSearchProvider.loadSearchResult(YoutubeSearchProvider.java:71)
	... 14 common frames omitted
```

I've uploaded the console log to pastebin:
[https://pastebin.com/0ThBSuha](url)

I did have a look to see if I could find any issues relating to this, but I couldn't find  anything relating to 1.3.50 - Sorry in advance if I've over looked any relatable issues.

**Comment by MCOfficer** [comment]
Dupes #501, fixed by #495. When searching for generic terms, results are more likely to be livestreams, which screw up the parsing.

**Comment by MCOfficer** [comment]
Can this be closed?
